### PR TITLE
Removing old app, updating Key Vault to T2 package.

### PR DESCRIPTION
**Pull request by FrankieTF**
Updated target:
- akvdotnet(Removed, since key-vault-console-app is this sample using T2 SDK.)
- key-vault-dotnet-core-quickstart(Updated Key Vault to T2 SDK)

### CODE REVIEW

**Comment by jongio** [comment]
@msmbaldwin - Can you please review?

**Comment by jongio** [comment]
@pakrym - Can you please review?

File: README.md [hunk 1]
```diff
@@ -103,12 +103,31 @@ This command shows the secret information including the URI. After completing th
 
     Open Program.cs file and update the placeholder <YourKeyVaultName> with the name of your Key Vault that you created in Step 3
 
-#### 7) Run the app
+#### 7) Set the environment variable for DefaultAzureCredential
+Follow one of the examples below depending on your operating system to create the environment variable. If using Windows close your open IDE or shell and restart it to be able to read the environment variable.
+
+Linux
```
Comment (pakrym) [inline README.md]: We should replace these with `az login` or `Connect-AzAccount` callouts. It's way simpler for customers than figuring out where to get credentials from.

**Review by pakrym — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (FrankieTF) [reply README.md]: Step 1 already called `az login`. So only removing this details for setting environment variable.

**Comment by JasonWHowell** [comment]
This PR broke Azure-docs-pr (our live docs). 

There are docs that rely upon the akvdotnet sample and our builds are now failing. 

https://docs.microsoft.com/en-us/azure/key-vault/secrets/quick-create-net-v3
CC: @msmbaldwin

**Comment by msmbaldwin** [comment]
Thank @JasonWHowell. I putting a fix in place: https://github.com/MicrosoftDocs/azure-docs-pr/pull/134520

**Comment by jongio** [comment]
I'll revert for now while we flesh this out.

**Comment by jongio** [comment]
Reverted this PR.  Apologies, we didn't realize this was tied to docs.  Will put better checks in place for future and sync with @msmbaldwin

**Pull request merged**